client/dashboard/app-dotcom/index.tsx

```diff
@@ -3,6 +3,7 @@ import Logo from './logo';
 import './style.scss';
 
 boot( {
+	instanceType: 'dotcom',
```

Comment:
I'm trying to think of other cases where this variable would be useful other than introduce hacks and poor decisions down the line (e.g., enable featured if `instanceType === instance` instead of using `supports` for example.

What if we explicitly passed `appName` instead? That way we can avoid the entropy :)

Reply:
My first solution was providing an app name, like "WordPress.com", as a service name.
But then, I figured out that I should provide default nameservers based on the instanceType.

Not sure what could be the best solution for that?

Reply:
>But then, I figured out that I should provide default nameservers based on the instanceType.

Are we doing that? AFAIK A4A isn't a domain registrar so it wouldn't have nameservers, right?

Reply:
Yeah - I also think this should be product name. We'll refactor the API endpoint to eventually return if the current name servers are the default ones or not so then none of that logic would be needed

Reply:
I'll change it. What would be the best config name?
- serverName
- productName
- name

What do you suggest?